src/library/F3DOptions.cxx

```diff
@@ -18,6 +18,12 @@
 #include <utility>
 #include <vector>
 
+//----------------------------------------------------------------------------
+F3DOptions::F3DOptions()
+{
+  this->Scalars = f3d::ReservedString; // XXX this should be removable at some point
```

Comment:
what is the point of moving that here ?

Reply:
`f3d::ReservedString` is unknown by the application. When instantiating `F3DOptions` at application level, it produces link error.

Reply:
F3DOptions will go into the application anyway. f3d_config.h should be installed.